docs/mobile/implement-event.mdx

```diff
@@ -88,10 +88,10 @@ public IEvent<SecurityCameraArgs> IntruderDetectedEvent => GetEvent<SecurityCame
 
 ## Using the Event
 
-Since the `CustomEvents` class was registered with autofac as a singleton, any other component, service, or view that injects it can publish or subscribe to the event. The example below but demonstrates how events could be used to tell views and services that rely on shared data that it has changed.
+Since the `CustomEvents` class was registered with autofac as a singleton, any other component, service, or view that injects it can publish or subscribe to the event. The example below demonstrates how events could be used to tell views and services that rely on shared data that it has changed.
```

Comment:
We should capitalize this autofac too if we are capitalizing the earlier one

Reply:
Capitalized.